website/static/js/fangorn.js

```diff
@@ -479,6 +481,11 @@ function doItemOp(operation, to, from, rename, conflict) {
         }
         // no need to redraw because fangornOrderFolder does it
         orderFolder.call(tb, from.parent());
+
+        if (operation === OPERATIONS.MOVE) {
```

Comment:
Could you use the webhook that we get back from waterbutler? Looks like there's a signal, too: https://github.com/CenterForOpenScience/osf.io/blob/03daf52befe01e62370ba82d4b239f89f161a482/website/addons/base/views.py#L382